spec/services/brewery_search_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe BrewerySearch do
+  describe ".find_brewery" do
+    it "returns the first brewery with the name sent in the search parameter" do
+      brewery_name = "Anchor"
+
+      stub_request(:any, "https://brianiswu-open-brewery-db-v1.p.rapidapi.com/breweries/search?query=#{brewery_name}").
+        with(
+          headers: {
+            "Host" => "brianiswu-open-brewery-db-v1.p.rapidapi.com:443",
+            "X-Rapidapi-Host" =>
+            "brianiswu-open-brewery-db-v1.p.rapidapi.com",
+            "X-Rapidapi-Key" =>
+            "d06411ee49mshcbcdc80cdd60989p1f0313jsn80c1ad5cec44",
+          },
+        ).to_return(status: 200,
+                    body: fake_json_response,
+                    headers: {})
+
+      result = described_class.find_brewery(brewery_name)
+
+      expect(result).to include(brewery_name)
+    end
+  end
+
+  def fake_json_response
+    "[{\"id\":314,\"name\":\"Anchor Brewing Co\",\"brewery_type\":\"large\",
```

Comment:
You may prefer writing the json as a ruby hash and converting it to json:

```ruby
[{id: 314, name: "Anchor Brewing Co"}].to_json
```

Reply:
This is on point.👌 Thanks!